include/sound/sof/topology.h

```diff
@@ -39,6 +39,8 @@ enum sof_comp_type {
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
```

Comment:
@keyonjie change per se looks good to me, after we address the backward compatibility issues. Is there any documentation about how do we keep backward compatibility?

E.g:
- newer kernels should work with older FW if the major number is the same
- newer FW should work with older kernels if the major number is the same

Reply:
We don't have documentation about this, per my understanding, @plbossart mentioned that we should keep backward compatible as the 1st item you mentioned as user might keep updating kernel while firmware usually keep intact:
"newer kernels should work with older FW if the major number is the same"

we have a kconfig item CONFIG_SND_SOC_SOF_STRICT_ABI_CHECKS and newer FW will be rejected when the item is selected.